common.h

```diff
@@ -32,85 +32,73 @@
 #include "arch/arch_device_api.h"
 
 #ifdef DEBUG_SOLVERS
-#define CHECK_FLOAT(x) \
-   if ((x) != (x)) {\
-      std::cerr << __FILE__ << ":" << __LINE__ << " Illegal value: " << x << std::endl;\
-      abort();\
+#define CHECK_FLOAT(x)                                                                                                                                                                                   \
+   if ((x) != (x)) {                                                                                                                                                                                   \
+      std::cerr << __FILE__ << ":" << __LINE__ << " Illegal value: " << x << std::endl;                                                                                                                \
+      abort();                                                                                                                                                                                         \
    }
 #else
-#define CHECK_FLOAT(x) {}
+#define CHECK_FLOAT(x)                                                                                                                                                                                 \
+   {                                                                                                                                                                                                   \
+   }
```

Comment:
This is a weird way to reformat this. It seems to identify the "{}" here as an empty code block, but this totally breaks the ideomatic use of {} to signify an empty piece of code.

Also, why does it even touch this line, if the regex forbids it from touching anything with a # in it?